lib/rust/ensogl/core/src/timer/timeout.rs

```diff
@@ -0,0 +1,84 @@
+//! FRP bindings for browser `setTimeout` timer.
```

Comment:
provide better explanation always, this should always describe a use case + maybe short example. either here or in the struct docs.